nextflow.config

```
@@ -148,6 +148,7 @@ profiles {
     test_bcl2fastq   { includeConfig 'conf/test_bcl2fastq.config'   }
     test_bclconvert  { includeConfig 'conf/test.config'             }
     test_full        { includeConfig 'conf/test_full.config'        }
+    test_sgdemux     { includeConfig 'conf/test_sgdemux.config'     }
```

```suggestion
    test_sgdemux     { includeConfig 'conf/test_sgdemux.config'     }    
    test_fqtk              { includeConfig 'conf/test_fqtk.config'               }
```

Can't get the formatting right on GitHub without a monospace font 🙃